docker-compose.yml

```diff
@@ -15,16 +15,16 @@ services:
       - fullstack
 
   booksdb_db_mysql:
-    image: mysql:8.0
+    image: mysql:latest
     container_name: booksdb_db_mysql
     ports:
-    - 3306:3306
+    - 3307:3306
     restart: on-failure
     environment:
-      - MYSQL_USER=vitalis
+      - MYSQL_USER=pavel
       - MYSQL_PASSWORD=password
-      - MYSQL_ROOT_PASSWORD=rootpassword
-      - MYSQL_DATABASE=booksdb
+      - MYSQL_ROOT_PASSWORD=Mysql&paha100688
```

Comment:
:)

Reply:
It's my config, Vitaliy should put his values